db/1.9/modules/ROOT/edits/database-connector-query-single.adoc

```diff
@@ -0,0 +1,215 @@
+
+The following example shows how to configure the Anypoint Connector for Database (Database Connector) *Query single* operation in a Mule application that retrieves books information from a table in a MySQL database. +
+
+To accomplish this example, you must: +
+
+* Run an SQL statement that creates a table called `BOOK` and inserts seven records of book information (title, author, bookshopName, isbn, price).
+* Configure the SQL database connection.
+* Create the Mule application.
+* Run, and test the application with curl commands.
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+.Query a Single Record from a Database flow
+image::database-query-single-flow.png[Query a Single Record from a Database flow]
+
+== Create the Table in the Database
+
+Before configuring the SQL database connection and creating the Mule app, run the following SQL statement to create a table called `BOOK` and inserts seven book records. An incremental `id` is associated to each new book added into the table. For example, the first book added into the table has an `id` equal to `1`, the second book will have an `id` equal to `2`, and so on.
+
+[source,sql,linenums]
+----
+USE SYS;
+
+CREATE TABLE BOOK(
+    id MEDIUMINT NOT NULL AUTO_INCREMENT,
+    title VARCHAR(100),
+    author VARCHAR(100),
+    bookshopName VARCHAR(100),
+    isbn VARCHAR(13),
+    price SMALLINT,
+    PRIMARY KEY (id)
+);
+
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('Good Omens', 'Terry Pratchett and Neil Gaiman', 'A.Z. Fell and Co.', '9780060853983', 50);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('A Wizard of Earthsea', 'Ursula K. Le Guin', 'A.Z. Fell and Co.', '9780547773742', 20);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('IT', 'Stephen King', 'A.Z. Fell and Co.', '9781508297123', 20);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('The Nice and Accurate Prophecies of Agnes Nutter', 'Agnes Nutter', 'A.Z. Fell and Co.', '000000000000', 200);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('Cujo', 'Stephen King', 'A.Z. Fell and Co.', '9781501192241', 20);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('Nation', 'Terry Pratchett', 'A.Z. Fell and Co.', '9780552557795', 30);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('The Ocean at the End of the Lane', 'Neil Gaiman', 'A.Z. Fell and Co.', '9780062459367', 30);
+----
+
+== Configure a MySQL Database Connection
+
+After you create the table, go to Anypoint Studio to configure the MySQL database connection:
+
+. Create a new Mule project in Studio.
+. Navigate to the *Global Elements* view.
+. Click *Create* to open the *Choose Global Type* view.
+. In the *Filter* field, type `http`, select *HTTP Listener config*, and click *OK*. +
+This configuration is for the HTTP *Listener* source that initiates the Mule application flow.
```

Comment:
```suggestion
+
These actions configure the HTTP *Listener* source that initiates the Mule application flow.
```

Reply:
GLOBALLY insert the + linespace between list item and description in all list items that are followed by additional text.